pkg/storaged/fsys-tab.jsx

```diff
@@ -0,0 +1,227 @@
+                                Value: old_opts,
+                                visible: function (vals) {
+                                    return vals.mounting == "custom";
+                                }
+                              }
+                          ],
+                          Action: {
+                              Title: _("Apply"),
+                              action: function (vals) {
+                                  return maybe_update_config(vals.mounting == "custom",
+                                                             vals.mount_point, vals.mount_options);
+                              }
+                          }
+            });
+        }
+
+        function mount () {
+            return block_fsys.Mount({});
+        }
+
+        function unmount () {
+            return block_fsys.Unmount({});
+        }
+
+        // See format-dialog.jsx for why we don't offer editing
+        // fstab for the old UDisks2
+
+        return (
+            <div>
+                <div className="tab-actions">
+                    <FormatButton client={this.props.client} block={this.props.block}/>
+                </div>
+                <table className="info-table-ct">
+                    <tr>
+                        <td>{_("Name")}</td>
+                        <td>
+                            <StorageLink onClick={rename_dialog}>
+                                {this.props.block.IdLabel || "-"}
+                            </StorageLink>
+                        </td>
+                    </tr>
+                    { !self.props.client.is_old_udisks2 ?
+                    <tr>
+                        <td>{_("Mount Point")}</td>
+                        <td>
+                            <StorageLink onClick={mounting_dialog}>
+                                {old_dir || _("(default)")}
+                            </StorageLink>
+                            { !is_filesystem_mounted ?
+                              <StorageButton onClick={mount}>{_("Mount")}</StorageButton>
+                              : null
+                            }
+                        </td>
+                    </tr> : null
+                    }
+                    { old_opts?
+                    <tr>
+                        <td>{_("Mount Options")}</td>
+                        <td>
+                            <StorageLink onClick={mounting_dialog}>
+                                {old_opts}
+                            </StorageLink>
+                        </td>
+                    </tr> : null
+                    }
+                    { (mounted_at.length > 0 || self.props.client.is_old_udisks2) ?
+                    <tr>
+                        <td>{_("Mounted At")}</td>
+                        <td>
+                            {mounted_at.join(", ")}
+                            { mounted_at.length > 0 ?
+                              <StorageButton onClick={unmount}>{_("Unmount")}</StorageButton> :
```

Comment:
According to the [coding guidelines](https://github.com/cockpit-project/cockpit/wiki/Cockpit-Coding-Guidelines#javascript-style), we want to wrap multiline jsx in parentheses. Now, this is a case of nested jsx...
In any case, I think the indentation needs to change - according to [Crockford](http://javascript.crockford.com/code.html) because of the operator `?` and since this is inside `{ }`.